crates/dc_bundle/src/legacy_definition/element/path.rs

```diff
@@ -75,3 +75,171 @@ impl Default for Stroke {
         }
     }
 }
+
+// This is a simple vector representation that should reproduce Figma or
+// another design tool's rendering. It simplifies the renderer's task by
+// precomputing the stroke path, and using the same definitions as the
+// toolkit schema.
+//
+// The serialized form should be somewhat compact and fast to deserialize
+// because we're storing different types (commands, values) in big arrays,
+// reducing the number of objects that need to be constructed during
+// deserialization.
+
+#[repr(u8)]
+#[derive(Clone, Copy, PartialEq, Debug)]
+pub enum PathCommand {
```

Comment:
PathCommand and some of the others here aren't serialized, but you think it makes sense to move to `dc_bundle`?

Reply:
These types or so tightly bound to the other ones here that use them that it made sense to keep them here. These are not serialized but they're converted to u8s that are then serialized.